guava/src/com/google/common/collect/Lists.java

```diff
@@ -527,7 +529,8 @@ public E get(int index) {
    * java.util.stream.Stream#map}. This method is not being deprecated, but we gently encourage you
    * to migrate to streams.
    */
-  public static <F, T> List<T> transform(
+  @DefaultQualifier(value = Nullable.class, locations = TypeUseLocation.UPPER_BOUND)
```

Comment:
Don't use this just to save a couple annoattions; write the couple annotations explicitly, if indeed any are needed.

Reply:
In this particular case, I think the annotations are already in place in the diff and just removing the `@DefaultQualifier` annotation will suffice.

Reply:
That is correct.